reporter/log.go

```diff
@@ -32,6 +32,10 @@ type logReporter struct {
 	logger *log.Logger
 }
 
+func (lr *logReporter) SendLog(logData go2sky.ReportedLogData) {
+
```

Comment:
follow trace to output the log to the console

Reply:
fixed